src/alloc.h

```diff
@@ -11,7 +11,8 @@
  * Used by test_alloc_fail() to test memory allocation failures.
  * Each entry must be on exactly one line, GetAllocId() depends on that.
  */
-typedef enum {
+typedef enum
+{
```

Comment:
This style change makes some tests fail.
I also think it's better to revert the style change of `struct` and not to use a separate line for `{`.
Some parts of the Vim source code and tests are auto-generated.

Reply:
> This style change makes some tests fail.

I think it is better to change the test at this point

> I also think it's better to revert the style change of struct and not to use a separate line for {.
Some parts of the Vim source code and tests are auto-generated.

Can you point out what part of the vim source code are auto generated? So maybe we can disable clang-format on those.